packages/plugin-commands-audit/src/audit.ts

```diff
@@ -171,6 +171,18 @@ export async function handler (
         output: err.message,
       }
     }
+
+    // Handling 404 response from audit API request
+    if (err instanceof AuditEndpointNotExistsError) {
+      let outputMessage = err.message
+      if (err.hint) {
+        outputMessage += `\n${err.hint}`
+      }
+      return {
+        exitCode: 1,
+        output: outputMessage,
+      }
+    }
```

Comment:
```suggestion
    throw err
```

You should just throw the error. pnpm will exit with 1 and print the error correctly